Directory.Packages.props

```
@@ -97,6 +99,6 @@
     <PackageVersion Include="xunit.runner.visualstudio" Version="2.8.2" />
     <PackageVersion Include="Yarp.ReverseProxy" Version="2.2.0" />
     <PackageVersion Include="IdentityModel" Version="7.0.0" />
-    <PackageVersion Include="Scalar.AspNetCore" Version="1.2.44"/>
+    <PackageVersion Include="Scalar.AspNetCore" Version="1.2.36" />
```

```suggestion
    <PackageVersion Include="Scalar.AspNetCore" Version="1.2.44" />
```

This is undoing a recent depend-a-bot update and it looks unrelated. Can we revert this change?